products/runtimeconfig/api.yaml

```diff
@@ -27,7 +27,7 @@ apis_required:
 objects:
   - !ruby/object:Api::Resource
     name: 'Config'
-    base_url: projects/{{project}}/configs/{{name}}
```

Comment:
Just curious, but any ideas why this didn't break TF (or even trigger a change on TF)?

Reply:
TF defaults the URL to have /{{name}} appended to the end if it ends in a non-variable.

Also TF doesn't generate this resource, only the IAM resource which also does some defaulting/normalization